fearless/Modules/AddAccount/Wireframes/AddAccount+UsernameSetupWireframe.swift

```diff
@@ -2,7 +2,11 @@ import Foundation
 
 extension AddAccount {
     final class UsernameSetupWireframe: UsernameSetupWireframeProtocol {
-        func proceed(from view: UsernameSetupViewProtocol?, model: UsernameSetupModel) {
+        func proceed(
+            from view: UsernameSetupViewProtocol?,
+            flow _: AccountCreateFlow = .wallet,
```

Comment:
looks like flow parameter unused ?

Reply:
@bnsports this is protocol method, sorry

Reply:
yep. we don't use it here, but protocols require it